protected/controllers/GpController.php

```diff
@@ -108,14 +108,21 @@ public function performGpSave(Contact $contact, Gp $gp, $isAjax = false)
                         $this->redirect(array('view', 'id' => $gp->id));
                     }
                 } else {
+                    if ($isAjax) {
```

Comment:
You could use Yii::app()->getRequest()->getIsAjaxREquest() instead of passing parameters around (see http://www.yiiframework.com/doc/api/1.1/CHttpRequest#getIsAjaxRequest-detail)